SonarQube.Common/AnalysisConfig/AnalysisConfig.cs

```diff
@@ -45,6 +45,9 @@ public class AnalysisConfig
 
         #region Serialization
 
+        [XmlIgnore]
+        public string FileName { get; private set; }
+
```

Comment:
Unrelated change: tweaked the AnalysisConfig class so it remembers where it was loaded from/saved to. Makes writing tests easier.